loops.js

```diff
@@ -18,7 +20,11 @@
 // klaar!
  // ==========================================
 
+for (let i=0; i<4; i++) {
+    console.log("loop...");
+}
 
+console.log("klaar!")
 
```

Comment:
In principe loop je nu 4x in plaats van 5x. Je zou nog een extra loop kunnen toevoegen en vervolgens door middel van een if-statement/conditie bepalen wat er gelogd wordt. Voor de uitkomst maakt het overigens niet uit :)